python/paddle/v2/fluid/layers.py

```diff
@@ -1,19 +1,31 @@
-import core
+import contextlib
+
 import proto.framework_pb2 as framework_pb2
+import core
 from framework import OpProtoHolder, Variable, Program, Operator
 from initializer import Constant, Normal, Xavier, Initializer
 from paddle.v2.fluid.layer_helper import LayerHelper, unique_name
-import re
-import cStringIO
+from registry import layer_registry
 from param_attr import ParamAttr
-import contextlib
 
 __all__ = [
     'fc', 'data', 'cross_entropy', 'conv2d', 'pool2d', 'embedding', 'concat',
     'StaticRNN', 'cast', 'sequence_conv', 'sequence_pool', 'sums', 'cos_sim',
     'batch_norm', 'accuracy', 'split_lod_tensor', 'While'
 ]
 
+layer_registry('mean')
+layer_registry('mul')
+layer_registry('elementwise_add')
+layer_registry('elementwise_div')
+layer_registry('dropout')
+layer_registry('reshape')
+layer_registry('sigmoid')
+layer_registry('scale')
+layer_registry('reshape')
+layer_registry('transpose')
+layer_registry('sigmoid_cross_entropy_with_logits')
```

Comment:
The name should be `register_layer`, it's more meaningful.

Reply:
fixed